proof/refine/ARM/TcbAcc_R.thy

```diff
@@ -315,6 +315,16 @@ lemma valid_tcbs'_maxPriority:
 
 lemmas valid_objs'_maxPriority = valid_tcbs'_maxPriority[OF valid_objs'_valid_tcbs']
 
+lemma valid_tcbs_valid_tcbE':
```

Comment:
Under that name, I'd expect a rule that eliminates `valid_tcbs'` and gives you `valid_tcb'`. Maybe `valid_tcbs_obj_at'` instead?